tests/cover/test_sampled_from.py

```diff
@@ -38,3 +42,8 @@ def test_cannot_sample_sets_in_strict_mode():
 def test_can_sample_sets_while_deprecated():
     with settings(strict=False):
         assert sampled_from(set('abc')).example() in 'abc'
+
+
+@given(sampled_from(an_enum))
+def test_can_sample_enums(member):
+    assert isinstance(member, an_enum)
```

Comment:
Done - if there's nothing else, I'd appreciate an approval so I can juggle some merges before going to bed.